test/conformance/ingress/path_test.go

```diff
@@ -212,7 +212,7 @@ func TestPathAndPercentageSplit(t *testing.T) {
 	const (
 		total     = 100
 		totalHalf = total / 2
-		tolerance = total * 0.15
+		tolerance = total * 0.20
```

Comment:
Should we increase the number of requests to do maybe? It might make sense to bump all these tests to 1000 requests and do them somewhat concurrently with `pool.go` maybe to save time?